src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+
+
+class Estimandizer:
+    """
+    Estimandizer. Generate estimands explicitly.
+    """
+
+    def __init__(self, data_handler, estimands):
+        self.data_handler = data_handler
+        self.estimands = estimands
+        self.transformations = []
+        self.transformation_map = {
+            "margin": [self.calculate_margin],
+            "voter_turnout_rate": [self.calculate_voter_turnout_rate],
+            "standardized_income": [self.standardize_median_household_income],
+            "age_groups": [self.create_age_groups],
+            "party_vote_share": [self.calculate_party_vote_share],
+            "education_impact": [self.calculate_party_vote_share, self.analyze_education_impact],
+            "gender_turnout_disparity": [self.investigate_gender_turnout_disparity],
+            "ethnicity_voting_patterns": [self.calculate_party_vote_share, self.examine_ethnicity_voting_patterns],
+            "income_impact": [
+                self.calculate_party_vote_share,
+                self.standardize_median_household_income,
+                self.explore_income_impact,
+            ],
+            "candidate": [self.candidate],
+        }
+
+    def check_estimand(self, estimand):
+        """
+        Ensure estimand isn't one of the pre-specified values that are already included
+        """
+        already_included = ["dem_votes", "gop_votes", "total_votes"]
+        if estimand in already_included:
+            return False
+        return True
+
+    def check_input_columns(self, columns):
+        """
+        Check that input columns contain all neccessary values for a calculation
+        """
+        missing_columns = [col for col in columns if col not in self.data_handler.data.columns]
+        return len(missing_columns) == 0
+
+    def predict_estimands(self, estimand):
```

Comment:
Can we change the name of this method? Maybe something like `verify_estimand` or `is_estimand_possible()`? Just cuz we're not really predicting anything.